packages/fluent/scss/treeview/index.scss

```
@@ -0,0 +1,26 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "treeview",
+    dependencies: (
+        "checkbox",
+        "icon"
+    )
+);
+
+
+// Component
+@forward "_variables.scss";
+@use "_layout.scss" as *;
+@use "_theme.scss" as *;
+
+// Register
+@use "../core/module-system" as module;
```

I should probably fix the generation script though 💯 

```suggestion
@use "../core/module-system/" as module;
```